dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -0,0 +1,223 @@
+            merged_profile.vocab = self.vocab.copy()
+            merged_profile._update_vocab(other.vocab)
+            
+        if "words" in merged_profile.__calculations:
+            merged_profile.word_count = self.word_count.copy()
+            for word in other.word_count:
+                if word in merged_profile.word_count:
+                    merged_profile.word_count[word] += other.word_count[word]
+                elif word not in self._stop_words:
+                    merged_profile.word_count[word] = other.word_count[word]
+                    
+        merged_profile.sample_size = self.sample_size + other.sample_size
+
+        return merged_profile
+
+    @property
+    def profile(self):
+        """
+        Property for profile. Returns the profile of the column.
+
+        :return:
+        """
+        profile = dict(
+            vocab=self.vocab,
+            words=list(self.word_count.keys()),
+            word_count=dict(self.word_count),
+            times=self.times,
+        )
+        return profile
+
+    @BaseColumnProfiler._timeit(name='vocab')
+    def _update_vocab(self, data, prev_dependent_properties=None,
+                      subset_properties=None):
+        """
+        Finds the unique vocabulary used in the text samples.
+
+        :param data: list or array of data from which to extract vocab
+        :type data: Union[list, numpy.array, pandas.DataFrame]
+        :param prev_dependent_properties: Contains all the previous properties
+            that the calculations depend on.
+        :type prev_dependent_properties: dict
+        :param subset_properties: Contains the results of the properties of the
+            subset before they are merged into the main data profile.
+        :type subset_properties: dict
+        :return: None
+        """
+        data_flat = list(itertools.chain(*data))
+        self.vocab = utils._combine_unique_sets(self.vocab, data_flat)
+
+    @BaseColumnProfiler._timeit(name='words')
+    def _update_words(self, data, prev_dependent_properties=None,
+                      subset_properties=None):
+        """
+        Finds the unique words and word count used in the text samples.
+
+        :param data: list or array of data from which to extract vocab
+        :type data: Union[list, numpy.array, pandas.DataFrame]
+        :param prev_dependent_properties: Contains all the previous properties
+            that the calculations depend on.
+        :type prev_dependent_properties: dict
+        :param subset_properties: Contains the results of the properties of the
+            subset before they are merged into the main data profile.
+        :type subset_properties: dict
+        :return: None
+        """
+        for word in list(" ".join(data).split(" ")):
+            if word in self.word_count:
+                self.word_count[word] += 1
+            elif word not in self._stop_words:
+                self.word_count[word] = 1
```

Comment:
```
if word not in self._stop_words:
    self.word_count[word] = 0
self.word_count[word] += 1
```

Reply:
This would make every valid word count equal to 1 and every non-valid word count equal to however many counts there are of it

Reply:
```python
if word not in self._stop_words:
    self.word_count[word] += 1
```
@lettergram no need to initialize first because it is a defaultdict(int)

Reply:
fixed